tasmota/language/fr_FR.h

```diff
@@ -139,10 +139,10 @@
 #define D_OK "Ok"
 #define D_ON "Marche"
 #define D_ONLINE "Connecté"
-#define D_ORP "ORP"
+#define D_ORP "Redox"                // Oxidation Reduction Potential
```

Comment:
Potentiel d'oxydoréduction
Redox applies to the reaction (Reaction d'Oxydoreduction)

Reply:
Too long. It's obvious for people that use that feature, they know what Redox is and it stays French.

Reply:
Agree